airgun/browser.py

```diff
@@ -186,7 +186,10 @@ def _get_selenium_browser(self):
             self._webdriver = webdriver.PhantomJS(
                 service_args=['--ignore-ssl-errors=true'])
         elif self.browser == 'remote':
-            self._webdriver = webdriver.Remote()
+            capabilities = settings.webdriver_desired_capabilities.__dict__
```

Comment:
Could you please describe `webdriver_desired_capabilities.__dict__`?
```python
In [1]: settings.webdriver_desired_capabilities
Out[1]:
{'build': 'Satellite 6.3.0-35.0',
 'idleTimeout': '1000',
 'maxDuration': '5400',
 'platform': 'macOS 10.12',
 'screenResolution': '1600x1200',
 'seleniumVersion': '2.53.1',
 'tags': '[automation-6.3-tier1-rhel7]',
 'tunnelIdentifier': 'automation-6_3-tier1-rhel7-1517743712690',
 'version': '47.0'}

In [2]: type(settings.webdriver_desired_capabilities)
Out[2]: dict

In [2]: settings.webdriver_desired_capabilities.__dict__
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
~/workspace/py3a/lib/python3.6/site-packages/manage/cli.py in <module>()
----> 1 settings.webdriver_desired_capabilities.__dict__

AttributeError: 'dict' object has no attribute '__dict__'
```
am i missing something?

Reply:
Yep, missing, that was command line output of robottelo's `settings.webdriver_desired_capabilities`, not airgun's. Please disregard